src/renderer/components/+catalog/details-for-catalog-entity.injectable.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import type { CatalogEntity } from "../../../common/catalog";
+import detailItemRegistrationsInjectable from "./detail-item-registrations.injectable";
+import { conforms, eq, includes } from "lodash/fp";
+
+const detailsForCatalogEntityInjectable = getInjectable({
+  instantiate: (di, catalogEntity: CatalogEntity) => {
+    const allRegistrations = di.inject(detailItemRegistrationsInjectable);
+
+    const registrationsForEntity = allRegistrations.get().filter(
+      conforms({
```

Comment:
TBH `lodash` is only library that I think we should depend without creating abstractions. That's because it provides helper functions which can be replaced relatively easily if ever needed. `lodash` provides a lot simplicity and ability to get rid nested code when combined with functional programming. (which we don't do at the moment)